scripts/i18n.js

```diff
@@ -84,4 +84,29 @@ exec('lingui extract', (error) => {
     }
     console.log(`${chalk.cyan('i18n')}: ${chalk.yellow('Lingui')} compiled messages`)
   });
-});
\ No newline at end of file
+});
+
+// Create crowdin.yml file
+const files = [];
+Object.values(icons).map(iconFolderName => {
+  files.push({
+    source: `/src/raw/${iconFolderName}/locales/en/messages.po`, // Path to the source file
+    dest: `icons/src/raw/${iconFolderName}/messages.po`, // Destination path in the repository
```

Comment:
Not sure about the destination path...But looking at how it's set up in [react](https://github.com/warp-ds/react/blob/next/crowdin.yml#L17) I'm thinking that this is correct?

Reply:
@siljec this looks perfectly fine 👏